crawler/config_parser.py

```diff
@@ -32,6 +32,11 @@ def apply_user_args(options={}):
         if 'avoid_setns' in options:
             crawlers[plugin]['avoid_setns'] = options['avoid_setns']
 
+        feature = plugin.replace('_host', '').replace('_container', '')
+        if feature in options:
+            for arg in options[feature]:
+                crawlers[plugin][arg] = options[feature][arg]
+
```

Comment:
but the plugin name can be anything, eg. package_new_host, shouln't we use get_features() plugin function?